### ISSUE TITLE
Main page is very large (~17MB)

### ISSUE THREAD

**Issue by jebej** [JuliaLang/www.julialang.org#54]
Here is a snip from the chrome web tools, sorted by size:

![image](https://user-images.githubusercontent.com/11141558/43644351-6da73790-96fc-11e8-88ce-02e3a02f6518.png)

**Comment by jebej** [comment]
Maybe some of these plots could be re-rendered with Plotly or similar.

**Comment by ViralBShah** [comment]
Maybe we should just use static images at least on mobiles?

**Comment by ViralBShah** [comment]
I also don't know how this affects our bandwidth bill with the CDN. I assume much of this just gets cached in the user's browser.

**Comment by mbauman** [comment]
What if we deferred loading the gifs until the tab is clicked?  We could even start with a static image and then dynamically swap it out once the gif arrives.  That should also help mitigate the bandwidth bill.

**Comment by mbauman** [comment]
Well after #60 we're down to 6.2MB, of which half is waves.gif.  Better, but still probably worth investigating deferred downloading or perhaps a more gif-optimization-friendly demo for the visualization tab.

**Comment by ViralBShah** [comment]
Or pick a different thing to show there. I just picked what I found on the Plots.jl pages.

**Comment by ranjanan** [comment]
Has #71 helped?

**Comment by KristofferC** [comment]
It is dominated by waves.gif but #72 at least took away a MB from it

**Comment by mbauman** [comment]
Here's after #72:

![screen shot 2018-08-03 at 5 19 59 pm](https://user-images.githubusercontent.com/154641/43668255-89db6946-9741-11e8-97d9-6c986ff55f90.png)

Why am I downloading some of the same scripts/css files three times?

**Comment by KristofferC** [comment]
17MB -> 5MB in 9 hours is good progress :)

**Comment by mbauman** [comment]
Ah, the triplets are coming from the three embedded YouTube videos.

**Comment by mbauman** [comment]
This looks promising to shave off ~1.5MB: https://www.labnol.org/internet/light-youtube-embeds/27941/

**Comment by ranjanan** [comment]
Nice. Will make a PR and test it

**Comment by essenciary** [comment]
I haven't tested it, but in theory, we could convert waves.gif to a video using something like http://gif-2-mp4.com

**Comment by ViralBShah** [comment]
I think we are now at 5MB. So probably ok to close this.

**Comment by KristofferC** [comment]
Emulating a slow 3g connection the website takes over 1 minute to load.

Fast 3g is 20 seconds.

I still think waves.gif is brutally too big.

**Comment by ViralBShah** [comment]
BTW, we can have any other plot. I think we should remove all the animated GIFs.

**Comment by jebej** [comment]
For reference, the home pages of other languages are generally below 1MB.
 * golang.org: 716KB
 * rust-lang.org: 495KB
 * python.org: 522 KB
 * r-project.org: 252 KB
